packages/babel-parser/test/fixtures/es2017/async-functions/invalid-generator-inside-loop/output.json

```diff
@@ -0,0 +1,47 @@
+{
+  "type": "File",
+  "start":0,"end":33,"loc":{"start":{"line":1,"column":0,"index":0},"end":{"line":1,"column":33,"index":33}},
+  "errors": [
+    "SyntaxError: Async functions can only be declared at the top level or inside a block. (1:10)"
```

Comment:
Currently Babel reports two errors, the first one is as above and the second one is from the generator *. Here we only report the first error.